Add password policy hint as comment in terraform.tfvars.example

While setting up Rancher using the quickstart on DigitalOcean for the first time, it failed because of the length of the password I provided. The specific error message is:

```
Error: [ERROR] Updating Admin password: [ERROR] Setting admin password: Bad response statusCode [422]. Status [422 Unprocessable Entity]. Body: [message=Password must be at least 12 characters, baseType=error, code=InvalidBodyContent] from [https://rancher.<IP address redacted>.sslip.io/v3/users/user-r9zs7?action=setpassword]
│
│   with module.rancher_common.rancher2_bootstrap.admin,
│   on ..\rancher-common\rancher.tf line 4, in resource "rancher2_bootstrap" "admin":
│    4: resource "rancher2_bootstrap" "admin" {
```

It would be nice if we could add a comment in terraform.tfvars.example to mention that the password length must be at least 12 characters. This probably applies to all providers since it's the Rancher admin password. I would add a comment like:

```
# - Must be at least 12 characters
```

If you prefer a PR from me, let me know.

--- a/aws/terraform.tfvars.example
+++ b/aws/terraform.tfvars.example
@@ -12,6 +12,7 @@ aws_secret_key = ""
 # aws_session_token = ""
 
 # Password used to log in to the `admin` account on the new Rancher server
+# - Must be at least 12 characters
 rancher_server_admin_password = ""
 
 # Add a windows node to the workload cluster

--- a/azure/terraform.tfvars.example
+++ b/azure/terraform.tfvars.example
@@ -15,6 +15,7 @@ azure_client_secret = ""
 azure_tenant_id = ""
 
 # Password used to log in to the `admin` account on the new Rancher server
+# - Must be at least 12 characters
 rancher_server_admin_password = ""
 
 # Add a windows node to the workload cluster

--- a/do/terraform.tfvars.example
+++ b/do/terraform.tfvars.example
@@ -6,6 +6,7 @@
 do_token = ""
 
 # Password used to log in to the `admin` account on the new Rancher server
+# - Must be at least 12 characters
 rancher_server_admin_password = ""
 
 

--- a/gcp/terraform.tfvars.example
+++ b/gcp/terraform.tfvars.example
@@ -10,6 +10,7 @@ gcp_account_json = ""
 gcp_project = ""
 
 # Password used to log in to the `admin` account on the new Rancher server
+# - Must be at least 12 characters
 rancher_server_admin_password = ""
 
 # Optional variables, uncomment to customize the quickstart

--- a/hcloud/terraform.tfvars.example
+++ b/hcloud/terraform.tfvars.example
@@ -6,6 +6,7 @@
 hcloud_token = ""
 
 # Password used to log in to the `admin` account on the new Rancher server
+# - Must be at least 12 characters
 rancher_server_admin_password = ""
 
 

--- a/linode/terraform.tfvars.example
+++ b/linode/terraform.tfvars.example
@@ -6,6 +6,7 @@
 linode_token = ""
 
 # Password used to log in to the `admin` account on the new Rancher server
+# - Must be at least 12 characters
 rancher_server_admin_password = ""
 
 

--- a/scw/terraform.tfvars.example
+++ b/scw/terraform.tfvars.example
@@ -12,6 +12,7 @@ scw_access_key = ""
 scw_secret_key = ""
 
 # Password used to log in to the `admin` account on the new Rancher server
+# - Must be at least 12 characters
 rancher_server_admin_password = ""
 
 

